src/node_http2.cc

```diff
@@ -938,6 +1004,30 @@ inline int Http2Session::OnFrameSent(nghttp2_session* handle,
                                      void* user_data) {
   Http2Session* session = static_cast<Http2Session*>(user_data);
   session->statistics_.frame_sent += 1;
+
+  int32_t id = GetFrameID(frame);
+  Http2Stream* stream = session->FindStream(id);
+  switch (frame->hd.type) {
+    case NGHTTP2_RST_STREAM:
+      TRACE_EVENT_NESTABLE_ASYNC_INSTANT2(
+          NODE_HTTP2_TRACING, "RST_STREAM",
```

Comment:
Is `RST_STREAM` an abbreviation for something more descriptive?  e.g., "Reset Stream"?

Reply:
Nope, it's literally "RST_STREAM" in the http2 spec.